gaps/gap-draft_gpu-pci-capability/gap-draft_gpu-pci-capability-proposal.md

```diff
@@ -0,0 +1,114 @@
+---
+gap: <to be assigned>
+title: GPU/PCI capability proposal
+description: Grammar Specifications for elements of GPU/PCI specification
+author: Stanislaw Krotki
+status: Draft
+type: <Experimental>
+---
+
+
+## Abstract
+GPU/PCI capability information in the Offer
+
+## Motivation
+This GAP introduces GPU (and possibly other PCI) devices capability description. Providers will specify this information so that Requestors would be able to specify certain processing requirements in Demands.
+
+## Specification
+
+* golem: {
+    * !exp: {
+        * gap-pci: { 
```

Comment:
Do we want to have `gap` in schema? Maybe `pci-device` or `device`, or `devices`?

Reply:
From my understanding - this is going to be updated to the gap number, once it is assigned, and will stay so as long as it's under experimental namespace.